src/sidebar/store/modules/test/selection-test.js

```diff
@@ -256,14 +256,14 @@ describe('sidebar/store/modules/selection', () => {
     it('should return `true` if focus enabled and valid `user` object present', () => {
       store = createStore(
         [selection],
-        [{ focus: { user: { username: 'whatever' } } }]
+        [{ focus: { user: { userid: 'acct:userid@authority' } } }]
       );
       assert.isTrue(store.focusModeHasUser());
     });
     it('should return `false` if focus enabled but `user` object invalid', () => {
       store = createStore(
         [selection],
-        [{ focus: { user: { displayName: 'whatever' } } }] // `username` is required
+        [{ focus: { user: { displayName: 'acct:userid@authority' } } }] // `userid` is required
```

Comment:
The previous value of `displayName` here was meant to be an example of a display name not a username or userid. Something like "Jim Smith" for example.

I try to use "realistic looking" examples for string values in tests as it provides better documentation about how the code is used in the real app.

Reply:
oops!